### ISSUE TITLE
Encrypted logic e2e test siltent failure

### ISSUE THREAD

**Issue by vimwitch** [Unirep/Unirep-Social#136]
<img width="1024" alt="image" src="https://user-images.githubusercontent.com/631020/198087493-b7d08de4-071d-4153-9eaf-344dc0ecea82.png">

The test is trying to login using the un-encrypted identity and is silently failing.

**Comment by AnthonyMadia** [comment]
Yes. This was mentioned on a PR when I submitted this. I believe this was because the semaphore identity was not being saved in local storage because we are only mocking the requests to the Twitter auth callbacks and not take taking the token out of the request and saving it to local storage with some `window.localStorage.setItem('privatekey', ......)` logic. 

An idea I had to resolve this was to do something similar to what is recommended in the cypress docs where they are making the request to the auth callback, taken the token out, and saving it to local storage like [this](https://docs.cypress.io/guides/end-to-end-testing/google-authentication#Custom-Command-for-Google-Authentication).
